powertools-sqs/pom.xml

```diff
@@ -54,12 +54,12 @@
             <artifactId>aws-lambda-java-events</artifactId>
         </dependency>
         <dependency>
-            <groupId>software.amazon.payloadoffloading</groupId>
-            <artifactId>payloadoffloading-common</artifactId>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>sqs</artifactId>
         </dependency>
         <dependency>
             <groupId>software.amazon.awssdk</groupId>
-            <artifactId>sqs</artifactId>
```

Comment:
This changing is a bit confusing, because it looks like the sqs dependency has been removed.

What is there a reason to remove the payloadoffloading dependency?

Reply:
Yeah change looks confusing but payload offloading is removed since we have the logic to parse payload point in repo itself, that lets us remove payload offload dep along with sdk v1 dep of s3 which came along with the library